elasticsearch_testing/gateways/learning-object-service-gateway.spec.js

```diff
@@ -0,0 +1,54 @@
+import { searchLearningObjects } from './learning-object-service-gateway';
+import { initElasticsearchNode } from './elasticsearch_gateway';
+import { expectedVisitorResponse, expectedEditorAdminResponse } from '../elasticsearch-data/elasticsearch-responses';
+import 'dotenv/config';
+
+describe('Sends a request to the learning object service search route', () => {
```

Comment:
Starting the describe statement with a word causes it to read oddly. Perhaps something as simple as:

```suggestion
describe('Learning Object search', () => {
```
or just a rewording of what is there now would help.